pipenv/routines/update.py

```diff
@@ -164,7 +160,7 @@ def upgrade(
             which=project._which,
             project=project,
             lockfile={},
-            category="default",
```

Comment:
This was definitely a bug in the update/upgrade path, it couldn't have worked too great for non-default categories.